..E..EE...E...E.......................
======================================================================
ERROR: test_adoptables_list_api_3 (__main__.MainTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests.py", line 31, in test_adoptables_list_api_3
    self.assertRegex(r.text, "(unauthorized key)")
AttributeError: 'MainTestCase' object has no attribute 'assertRegex'

======================================================================
ERROR: test_get_organizations_list_api_2 (__main__.MainTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests.py", line 123, in test_get_organizations_list_api_2
    self.assertRegex(r.text, "(unauthorized key)")
AttributeError: 'MainTestCase' object has no attribute 'assertRegex'

======================================================================
ERROR: test_get_organizations_list_api_3 (__main__.MainTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests.py", line 129, in test_get_organizations_list_api_3
    self.assertRegex(r.text, "(unauthorized key)")
AttributeError: 'MainTestCase' object has no attribute 'assertRegex'

======================================================================
ERROR: test_get_single_adoptable_api_3 (__main__.MainTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests.py", line 55, in test_get_single_adoptable_api_3
    self.assertRegex(r.text, "(unauthorized key)")
AttributeError: 'MainTestCase' object has no attribute 'assertRegex'

======================================================================
ERROR: test_get_single_breed_api_3 (__main__.MainTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests.py", line 101, in test_get_single_breed_api_3
    self.assertRegex(r.text, "(Invalid appid)")
AttributeError: 'MainTestCase' object has no attribute 'assertRegex'

----------------------------------------------------------------------
Ran 38 tests in 7.177s

FAILED (errors=5)
